input-github-username.sh

```diff
@@ -0,0 +1,11 @@
+set -u;
+if ! gp env | grep -q 'GITHUB_USERNAME'; then {
+  until read -r -p "++++ Enter your Github username (only required once): " GITHUB_USERNAME; do continue; done
+  gp env GITHUB_USERNAME="$GITHUB_USERNAME"; # Sets this variable on https://gitpod.io/variables which can be used later on
```

Comment:
Can you try also putting `export GITHUB_USERNAME="$GITHUB_USERNAME"` to see if it correctly applies the env var to the caller of this file?